### DISCUSSION TITLE
is there a method to check the validity of a field regardless of if it has been validated yet?

### DISCUSSION THREAD

**Discussion by IDrumsey** [react-hook-form/react-hook-form#11569] [Q&A]
I have a use case where my form uses the `onTouched` validation mode, but I need to check if one of my fields is valid before the form fields have been validated. Is there already something for this? If not it could be cool to have something like

```ts
const {register, checkIfFieldIsValid} = useForm({mode: "onTouched"})

const myCustomFunction = () => {
  if(checkIfFieldIsValid("myField")) {
    // do stuff
  }
}
```

With this type of thing, it would be possible to check if a field is valid even if `formState.errors.myField` isn't accurate due to the `useForm` `mode` being `onTouched`.

**Comment by Moshyfawn** [comment]
I think you want [the `trigger` method](https://react-hook-form.com/docs/useform/trigger).

**Reply by IDrumsey** [reply]
I don't want to _trigger_ the validation because then when my custom functionality runs it would cause error messages to show up on the UI when I don't want them to. To help elaborate, my problem would be fixed if I didn't use `onTouched` mode, but I want to use that. So the question is, how can I check if a field is valid without effecting the form state before the first actual form validation has been triggered.

**Comment by IDrumsey** [comment]
Here's an SO if that helps - https://stackoverflow.com/questions/78116060/react-hook-form-how-to-check-if-form-field-is-valid-before-form-validation-is-fi
